### PR TITLE
[Clang-Tidy] Add log messages

### PR DESCRIPTION
**Pull request by ybiquitous**
> Explain a summary, purpose, or background for this change.

To make debugging easier.

> Link related issues, e.g. "Fix #<ISSUE_ID>", "Related to #<ISSUE_ID>", or "None."

None.

> Check the following items.

- [] ~~Add a new [changelog](https://github.com/sider/runners/blob/master/CHANGELOG.md) entry if this change is notable.~~
  → Not notable change.

### CODE REVIEW

File: lib/runners/processor/clang_tidy.rb [hunk 1]
```diff
@@ -31,21 +31,27 @@ def analyzer_bin
...
+          analyzed_files << path
         end
 
-      Results::Success.new(guid: guid, analyzer: analyzer).tap do |result|
-        result.add_issue(*issues)
+      if analyzed_files.empty?
+        trace_writer.message "No files to analyze."
+      else
+        trace_writer.message "#{analyzed_files.size} file(s) were analyzed."
       end
```
Comment (suetsuguk) [inline lib/runners/processor/clang_tidy.rb:52]: [suggestion] a ternary operator gives you  just one line of code

```suggestion
      trace_writer.message analyzed_files.empty? ? "No files to analyze." : "#{analyzed_files.size} file(s) were analyzed."
```

Comment (ybiquitous) [reply lib/runners/processor/clang_tidy.rb:52]: Make sense. 👍

**Comment by suetsuguk** [comment]
LGTM. Thank you.

**Comment by ybiquitous** [comment]
Thanks for the review!

**Pull request merged**